### PR TITLE
Добавить поиск при выборе крипты на странице exchange

### PR DESCRIPTION
**Pull request by linuxweeva**
# Pull request template

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/swaponline/swap.react/wiki/CONTRIBUTING) guide
- [x] branch rebased on `develop`
- [x] styleguide check `npm run validate`
- [x] good naming
- [x] keep simple
- [x] video or screenshot attached
- [ ] testing instruction attached
- [x] check your PR once again

### Video or screenshot

https://drive.google.com/file/d/1vX_TT5LYLAwGyN-tHvsC0WgZyTtQjP8q/view

### CODE REVIEW

File: shared/components/ui/DropDown/DropDown.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (Pasha8914) [inline shared/components/ui/DropDown/DropDown.js]: ?

File: shared/components/ui/DropDown/DropDown.js [hunk 2]
```diff
@@ -39,16 +40,20 @@ export default class DropDown extends Component {
...
+      itemsFiltered:[],
+      searchValue:'',
     }
   }
 
   toggle = () => {
     const { isToggleActive, toggleOpen, toggleClose } = this.props
 
     if (isToggleActive) {
+      this.setState({
```
Comment (Pasha8914) [inline shared/components/ui/DropDown/DropDown.js]: ?

File: shared/components/ui/DropDown/DropDown.js [hunk 3]
```diff
@@ -109,12 +130,26 @@ export default class DropDown extends Component {
         <div styleName={dropDownStyleName} className={className}>
           <div styleName="selectedItem" onClick={this.toggle}>
             <div styleName="arrow" />
-            {this.renderSelectedItem()}
+            {isToggleActive ? (
+              <input styleName="searchInput" type="text" value={this.state.searchValue} autoFocus onChange={this.handleChangee} />
```
[the remark this answers is not in the document]
Comment (linuxweeva) [reply shared/components/ui/DropDown/DropDown.js]: Не понял про линк, а что именно?

[the remark this answers is not in the document]
Comment (nikdementev) [reply shared/components/ui/DropDown/DropDown.js]: image

**Pull request merged**